ansible/files/telemetry.sh

```diff
@@ -50,12 +50,6 @@ wget -nc https://3230d63b5fc54e62148e-c95ac804525aac4b6dba79b00b39d1d3.ssl.cf1.r
 bash Anaconda2-2.4.0-Linux-x86_64.sh -b
 $ANACONDAPATH/bin/pip install python_moztelemetry python_mozaggregator montecarlino runipy py4j==0.8.2.1 pyliblzma==0.5.3 plotly==1.6.16 seaborn==0.6.0
 
-# Force Python 2.7 (Python executable path seems to be hardcoded in Spark)
-sudo rm /usr/bin/python /usr/bin/pip
-sudo ln -s $ANACONDAPATH/bin/python /usr/bin/python
-sudo ln -s $ANACONDAPATH/bin/pip /usr/bin/pip
-sudo sed -i '1c\#!/usr/bin/python2.6' /usr/bin/yum
-
 # Add public key
```

Comment:
In my testing Spark actually fails to run if there is a version mismatch between master and worker nodes, but because of 
https://github.com/whd/emr-bootstrap-spark/blob/master/ansible/files/telemetry.sh#L82 the master runs the newer python.